doc/docs/master/deploy-manage/deploy/deploy-pachyderm-ide.md

```diff
@@ -0,0 +1,92 @@
+# Deploy the Pachyderm IDE
+
+!!! note
+    The Pachyderm IDE is an enterprise feature,
+    which is also available for testing during
+    the 14-day free trial.
+    Contact sales@pachyderm.com for more
+    information.
+
+The Pachyderm Integrated Development Environment (IDE) is
+an optional extension to Pachyderm clusters that provides a
+comprehensive environment for prototyping and deploying
+Pachyderm pipelines, as well as introspecting on data in the
+Pachyderm version control system. It combines together Jupyter,
+JupyterHub, and JupyterLab, which are familiar tools for many data
+scientists.
+
+The following diagram shows the Pachyderm IDE deployment.
+
+![JupyterHub and Pachyderm Architecture Overview](../../assets/images/d_jupyterhub-pachyderm-arch.svg)
+
+In the diagram above, you can see that Pachyderm and JupyterHub are
+deployed on the same Kubernetes cluster. You deploy Pachyderm by
+using the `pachctl deploy` command as described below. After
+deployment, you log in to JupyterHub with your Pachyderm user
+and interact with Pachyderm from within the JupyterLab UI by
+using `pachctl` or the Pachyderm Python client.
+
+## Prerequisites
+
+Before deploying Pachyderm IDE, configure the following prerequisites:
+
+* Deploy Pachyderm 1.11.0 or later as described in
+[Deploy Pachyderm](../../deploy-manage/deploy/)
+on a supported Kubernetes platforms:
+
+  - Google Kubernetes Engine (GKE)
+  - Amazon Elastic Container Service (EKS)
+  - Docker Desktop for Mac
+
+    For more information about JupyterHub requirements for Kubernetes,
+    see [Zero to JupyterHub with Kubernetes](https://zero-to-jupyterhub.readthedocs.io/en/latest/).
+
+* Register your Enterprise token as described in
+[Activate Pachyderm Enterprise Edition](../..//enterprise/deployment/#activate-pachyderm-enterprise-edition).
+
+* Enable [Pachyderm Access Controls](../../enterprise/auth/auth/).
+
+## Deploy Pachyderm IDE
+
+After you deploy Pachyderm and enable authentication,
+deploy the Pachyderm IDE by running:
+
+```bash
+pachctl deploy ide
+```
+
+Pachyderm deploys and configures JupyterHub and JupyterLab, which
+might take some time. 
+
+## Log in to Pachyderm IDE
+
+After you deploy the Pachyderm IDE, you can access the JupyterLab UI
+in a web browser through the Pachyderm cluster hostname on port
+`80`. To get the IP address of the Pachyderm IDE,
```

Comment:
1) This isn't quite true, as you're accessing the service IP rather than the cluster IP (they are different on cloud deployments.)
2) You're technically accessing the JupyterHub UI when you're on the login page.

I think something like this would work better:

```
After you deploy the Pachyderm IDE, you can access it in a
web browser through it's service IP on port 80.
```

Reply:
addressed